pegasus/sites.v3/code.org/public/accessibility.md

```diff
@@ -0,0 +1,75 @@
+---
+title: 'Accessibility'
+theme: responsive
+---
+
+# Code.org’s Commitment to Accessibility
+
+Code.org believes computer science is foundational for all students and we are committed to equity, access, and opportunity in our organizational [values](https://code.org/about/values). To achieve this goal, we work to identify and eliminate barriers that prevent the inclusion and full participation of students and educators with disabilities.
```

Comment:
You can use `/about/values` as a relative URL here, and then it will go to the page on the current environment.